card/spec/models/card_spec.rb

```diff
@@ -83,14 +83,14 @@ def assert_stable card1
   end
 
   it 'should remove cards' do
-    forba = Card.create! :name=>"Forba"
-    torga = Card.create! :name=>"TorgA"
-    torgb = Card.create! :name=>"TorgB"
-    torgc = Card.create! :name=>"TorgC"
+    forba = Card.create! name: "Forba"
+    torga = Card.create! name: "TorgA"
```

Comment:
Useless assignment to variable - `torga`. Did you mean `torgb`?<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.